index.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+
+const core = require('@actions/core'),
+    github = require('@actions/github'),
+    token = core.getInput('token'),
+    octokit = github.getOctokit(token);
+
+if(!github.context.payload.issue && !/\/(?:issue|pull-request)s\/\d+$/.test(github.context.payload.project_card?.content_url)) {
+    core.info('Not running on an event with an associated issue.');
+    return;
+}
+
+async function getIssue() {
+    let { issue } = github.context.payload;
+    if(!issue) {
+        const issueNumber = /\/(?:issue|pull-request)s\/(\d+)$/.exec(github.context.payload.project_card?.content_url);
+        const { data: result } = await octokit.rest.issues.get({
+            ...github.context.repo,
+            issue_number: issueNumber[1],
+        });
+        issue = result;
+    }
+    return issue;
+}
```

Comment:
I wonder if it would make sense to have at least the issue number passed in by the parser. That way only the parser depends on the context from the trigger payload.